src/locale/locales/es/messages.po

```diff
@@ -2639,41 +2639,41 @@ msgstr "Medios externos"
 #: src/view/com/modals/AddAppPasswords.tsx:119
 #: src/view/com/modals/AddAppPasswords.tsx:123
 msgid "Failed to create app password."
-msgstr ""
+msgstr "Error al crear la contraseña de la aplicación."
 
 #: src/screens/StarterPack/Wizard/index.tsx:229
 #: src/screens/StarterPack/Wizard/index.tsx:237
 msgid "Failed to create starter pack"
-msgstr ""
+msgstr "Error al crear el paquete inicial."
```

Comment:
```suggestion
msgstr "Error al crear el paquete de inicio."
```

It would be best to keep it consistent between "paquete inicial" and "paquete de inicio"